src/WebJobs.Script.WebHost/Diagnostics/DiagnosticEventLogger.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Azure.WebJobs.Script.Config;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Diagnostics
+{
+    public class DiagnosticEventLogger : ILogger
+    {
+        private const string ErrorCode = "errorCode";
+        private const string HelpLink = "helpLink";
```

Comment:
Can you can use these from `ScriptConstants`?

Reply:
done